### PR TITLE
Not enough points task

### PR DESCRIPTION
**Pull request by MKDEVELOPEMENT**
I have made the snackbar appear when user doent have enough points to purchase

### CODE REVIEW

File: PowerUp/app/build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 apply plugin: 'com.android.application'
 
+
```
Comment (chhavip) [inline PowerUp/app/build.gradle]: remove extra line

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 2]
```diff
@@ -538,63 +536,65 @@ public void onClick(View v) {
             public void onClick(View v) {
                 getmDbHandler().open();
                 if (value.equalsIgnoreCase(getResources().getString(R.string.cloth))) {
-                    if(SessionHistory.totalPoints < getmDbHandler().getPointsClothes(cloth))
-                        Toast.makeText(SelectFeaturesActivity.this, R.string.points_check, Toast.LENGTH_SHORT).show();
-                    else{
+                    if(SessionHistory.totalPoints < getmDbHandler().getPointsClothes(cloth)) {
+                        Snackbar bar = Snackbar.make(v, "Sorry, you did not have enough points for that purchas", Snackbar.LENGTH_LONG);
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Do not hard-code string but put in strings.xml

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 3]
```diff
@@ -538,63 +536,65 @@ public void onClick(View v) {
...
                         getmDbHandler().setAvatarCloth(cloth);
                         getmDbHandler().setPurchasedClothes(cloth);
                         SessionHistory.totalPoints = SessionHistory.totalPoints - getmDbHandler().getPointsClothes(cloth);
                     }
                 } else if (value.equalsIgnoreCase(getResources().getString(R.string.hair))) {
-                    if(SessionHistory.totalPoints < getmDbHandler().getPointsHair(hair))
-                        Toast.makeText(SelectFeaturesActivity.this, R.string.points_check, Toast.LENGTH_SHORT).show();
-                    else{
+                    if(SessionHistory.totalPoints < getmDbHandler().getPointsHair(hair)) {
+                        Snackbar bar = Snackbar.make(v, "Sorry, you did not have enough points for that purchase", Snackbar.LENGTH_LONG);
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Same, use the same string as above.

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 4]
```diff
@@ -538,63 +536,65 @@ public void onClick(View v) {
...
                             getmDbHandler().setPurchasedAccessories(glassesPurchased);
                             getmDbHandler().setAvatarGlasses(glasses);
                         }
                     }
                     if (bagPurchased != 0){
-                        if(SessionHistory.totalPoints < getmDbHandler().getPointsAccessories(bagPurchased))
-                            Toast.makeText(SelectFeaturesActivity.this, R.string.points_check, Toast.LENGTH_SHORT).show();
-                        else{
+                        if(SessionHistory.totalPoints < getmDbHandler().getPointsAccessories(bagPurchased)) {
+                            Snackbar bar = Snackbar.make(v, "Sorry, you did not have enough points for that purchase", Snackbar.LENGTH_LONG);
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Same

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 5]
```diff
@@ -538,63 +536,65 @@ public void onClick(View v) {
...
                             getmDbHandler().setPurchasedAccessories(bagPurchased);
                             getmDbHandler().setAvatarBag(bag);
                         }
                     }
                     if (necklacePurchased != 0){
-                        if(SessionHistory.totalPoints < getmDbHandler().getPointsAccessories(necklacePurchased))
-                            Toast.makeText(SelectFeaturesActivity.this, R.string.points_check, Toast.LENGTH_SHORT).show();
-                        else{
+                        if(SessionHistory.totalPoints < getmDbHandler().getPointsAccessories(necklacePurchased)) {
+                            Snackbar bar = Snackbar.make(v, "Sorry, you did not have enough points for that purchase", Snackbar.LENGTH_LONG);
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Same

File: PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java [hunk 6]
```diff
@@ -538,63 +536,65 @@ public void onClick(View v) {
...
                         getmDbHandler().setPurchasedHair(hair);
                         SessionHistory.totalPoints = SessionHistory.totalPoints - getmDbHandler().getPointsClothes(cloth);
                     }
                 } else if (value.equalsIgnoreCase(getResources().getString(R.string.accessory))) {
                     if (hatPurchased != 0){
-                        if(SessionHistory.totalPoints < getmDbHandler().getPointsAccessories(hatPurchased))
-                            Toast.makeText(SelectFeaturesActivity.this, R.string.points_check, Toast.LENGTH_SHORT).show();
-                        else{
+                        if(SessionHistory.totalPoints < getmDbHandler().getPointsAccessories(hatPurchased)) {
+                            Snackbar bar = Snackbar.make(v, "Sorry, you did not have enough points for that purchase", Snackbar.LENGTH_LONG);
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/SelectFeaturesActivity.java]: Same. Also give a more meaningful name to Snackbar instance, notEnoughtPointsBar for example.

**Comment by aanchal07** [comment]
Closing the pull request since there are conflicts.

**Pull request closed without merging**